web/src/lightbox.js

```diff
@@ -75,11 +75,15 @@ export class PanZoomControl {
             switch (e.key) {
                 case "Z":
                 case "+":
-                    this.zoomIn();
+                    if ($("#lightbox_overlay .image-preview img").length > 0) {
+                        this.zoomIn();
+                    }
                     break;
                 case "z":
                 case "-":
-                    this.zoomOut();
+                    if ($("#lightbox_overlay .image-preview img").length > 0) {
+                        this.zoomOut();
+                    }
```

Comment:
Can we check some variables in this module rather than checking HTML in the DOM? I feel like that would be much more maintainable.

Reply:
OK, Changed it.